nss-day-cohort-48/debugging-assessment-thejmdw
# debugged enough to render

To test:

1. Pull code
2. git checkout thejmdw-first-commit
3. open in VScode
4. serve and open in Chrome
5. check to see that html is being rendered to the DOM
